x86/efi/efistart64.S

```diff
@@ -16,3 +31,34 @@ load_idt:
 	lidtq idt_descr(%rip)
 
 	retq
+
+.globl load_gdt_tss
+load_gdt_tss:
+	/* Set GDT runtime address */
```

Comment:
This is not needed anymore. I will remove this in the next revision.

Reply:
Ack.